azure-communication-ui/azure-communication-ui-demo-app/src/main/res/layout/memory_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would suggest styling the Text with no background, small size. Make the text white and give it a heavy black shadow with offset (0,0), blur/spread of like 2.  Minimize padding and font size. That should make it highly readable while being out of the way and mostly translucent.

Maybe later/nice to have we can color text based on memory pressure?